Mlem/ContentView.swift

```diff
@@ -20,29 +20,30 @@ struct ContentView: View {
         TabView(selection: $tabSelection) {
             AccountsPage()
                 .tabItem {
-                    Label("Feeds", systemImage: "text.bubble")
+                    Label("Feeds", systemImage: "scroll")
+                        .environment(\.symbolVariants, tabSelection == 1 ? .fill : .none)
                 }.tag(1)
             
             if let currentActiveAccount = appState.currentActiveAccount {
                 InboxView(account: currentActiveAccount)
                     .tabItem {
                         Label("Inbox", systemImage: "mail.stack")
+                            .environment(\.symbolVariants, tabSelection == 2 ? .fill : .none)
                     }.tag(2)
                 
                 NavigationView {
                     ProfileView(account: currentActiveAccount)  
                 } .tabItem {
-                    if showUsernameInNavigationBar {
-                        Label(currentActiveAccount.username, systemImage: "person")
-                    } else {
-                        Label("Profile", systemImage: "person")
-                    }
+                    let username = showUsernameInNavigationBar ? currentActiveAccount.username : "Profile"
```

Comment:
Can we refactor code like this into a function or computed property, outside of the body?